shared/wallets/send-form/note-and-memo/index.stories.js

```diff
@@ -4,17 +4,34 @@ import * as Sb from '../../../stories/storybook'
 import NoteAndMemo from '.'
 
 const commonProps = {
+  publicMemo: '',
+  secretNote: '',
   onChangePublicMemo: Sb.action('onChangePublicMemo'),
   onChangeSecretNote: Sb.action('onChangeSecretNote'),
 }
 const load = () => {
   Sb.storiesOf('Wallets/SendForm/Note and Memo', module)
     .add('Normal input', () => <NoteAndMemo {...commonProps} />)
     .add('Input with a note error', () => (
-      <NoteAndMemo {...commonProps} noteError="There's something wrong with your note." />
+      <NoteAndMemo {...commonProps} secretNoteError="There's something wrong with your note." />
     ))
     .add('Input with a memo error', () => (
-      <NoteAndMemo {...commonProps} memoError="There's something wrong with your memo." />
+      <NoteAndMemo {...commonProps} publicMemoError="There's something wrong with your memo." />
+    ))
+    .add('Prefilled', () => (
+      <NoteAndMemo
+        {...commonProps}
+        secretNote="This is a prefilled secret note"
+        publicMemo="This is a prefilld pblc memo"
```

Comment:
thes wods pelt rong :)

Reply:
hah yeah, i was surprised by how short the public memo limit is in practice